pycbc/inference/sampler/emcee_pt.py

```diff
@@ -114,13 +122,34 @@ def from_config(cls, cp, model, nprocesses=1, use_mpi=False):
             "name in section [sampler] must match mine")
         # get the number of walkers to use
         nwalkers = int(cp.get(section, "nwalkers"))
-        # get the number of temps
-        ntemps = int(cp.get(section, "ntemps"))
+        if cp.has_option(section, "ntemps") and \
+                cp.has_option(section, "inverse-temperatures-input-file"):
+            raise ValueError("Must specify either ntemps or "
+                             "inverse-temperatures-input-file, not both.")
+        if cp.has_option(section, "inverse-temperatures-input-file"):
+            # get the path of the file containing inverse temperatures values
+            # to be provided to emcee_pt for the temperature ladder. This
+            # should be an hdf file having an attribute named 'betas' storing
+            # the list of inverse temperature values.
```

Comment:
Thanks for adding this comment, but I you should add it to the doc string for the function, so it shows up in the online documentation.